robots/pkg/kubevirtci/kubevirtci_test.go

```diff
@@ -222,6 +221,9 @@ func TestBumpMinorReleaseOfProvider(t *testing.T) {
 				t.Errorf("BumpMinorReleaseOfProvider() error = %v, wantErr %v", err, tt.wantErr)
 			}
 			got, err := ReadExistingProviders(dir)
+			if err != nil {
+				panic(err)
```

Comment:
I'd prefer to raise an error instead of a panic here.

```suggestion
				t.Errorf("ReadExistingProvider raised unexpected error = %v", err)
```